annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
The reason that the sample rate must be between 1 and 100 is because the sample rate function is something like: `f(sampleRate, NumberOfSample) = NumberOfSamples*(sampleRate/100)` so in the statful client context having a sampleRate of `0` is achieved by not adding the annotation on methods or not enabling the client.

I think enabling/disabling the aspect doesn't makes sense as we should just not use the annotation if we don't want to measure the method.

I think the `-1` default with proper javadoc might achieve what we want in a nice way.